sim/midas/src/main/scala/midas/models/dram/Util.scala

```diff
@@ -410,9 +410,9 @@ class FIFOAlignedAddressMatcher(val entries: Int, addrWidth: Int, size: Int, len
 
   io.hit := addrs.exists({entry =>  entry.valid && 
```

Comment:
I know the syntax style is all over the place in FireSim, but i'm moving to the more scala-standard style of 
```
addrs.exists { entry =>
   <body of function at one level of indentation
}

Reply:
Done